tests/sqlparser_mssql.rs

```diff
@@ -55,6 +55,80 @@ fn parse_mssql_delimited_identifiers() {
     );
 }
 
+#[test]
+fn parse_create_procedure() {
+    let sql = "CREATE OR ALTER PROCEDURE test (@foo INT, @bar VARCHAR(256)) AS BEGIN SELECT 1 END";
+    assert_eq!(
+        ms().verified_stmt(sql),
+        Statement::CreateProcedure {
+            or_alter: true,
+            body: vec![Statement::Query(Box::new(Query {
+                with: None,
+                limit: None,
+                offset: None,
+                fetch: None,
+                locks: vec![],
+                order_by: vec![],
+                body: Box::new(SetExpr::Select(Box::new(Select {
+                    distinct: None,
+                    top: None,
+                    projection: vec![SelectItem::UnnamedExpr(Expr::Value(Value::Number(
+                        "1".to_string(),
+                        false
+                    )))],
+                    into: None,
+                    from: vec![],
+                    lateral_views: vec![],
+                    selection: None,
+                    group_by: vec![],
+                    cluster_by: vec![],
+                    distribute_by: vec![],
+                    sort_by: vec![],
+                    having: None,
+                    named_window: vec![],
+                    qualify: None
+                })))
+            }))],
+            params: Some(vec![
+                ProcedureParam {
+                    name: Ident {
+                        value: "@foo".into(),
+                        quote_style: None
+                    },
+                    data_type: DataType::Int(None)
+                },
+                ProcedureParam {
+                    name: Ident {
+                        value: "@bar".into(),
+                        quote_style: None
+                    },
+                    data_type: DataType::Varchar(Some(CharacterLength {
+                        length: 256,
+                        unit: None
+                    }))
+                }
+            ]),
+            name: ObjectName(vec![Ident {
+                value: "test".into(),
+                quote_style: None
+            }])
+        }
+    )
+}
+
+#[test]
+fn parse_mssql_create_procedure() {
```

Comment:
Can you also add a test for multiple statements? All these tests seem to have just a single one